apps/docs/src/guide/contracts/logs.md

```diff
@@ -6,7 +6,7 @@ Consider the following example contract:
 
 <<< @/../../docs-snippets/test/fixtures/forc-projects/log-values/src/main.sw#log-1{rust:line-numbers}
 
-To access the logged values in TypeScript, use the `logs` property in the `FunctionInvocationResult` of a contract call result. The logs data will be stored in an `Array<any>`:
+To access the logged values in TypeScript, use the `logs` property in the response of a contract call. The logs data will be stored in an `Array<any>`:
```

Comment:
See 

```suggestion
To access the logged values in TypeScript, use the `logs` property in the response of a contract call. The logs data will be stored in an `Array<any>`:
```

Reply:
Hey @maschad as @petertonysmith94 [mentioned](https://github.com/FuelLabs/fuels-ts/pull/2652#discussion_r1661919216) if we decided to move in that direction we would need to address this at another PR as it will involve changing other parts of the code.